src/lib/features/export-import-toggles/import-validation-messages.ts

```diff
@@ -92,6 +95,16 @@ export class ImportValidationMessages {
                 affectedItems: [...featureNameCheckResult.invalidNames].sort(),
             });
         }
+        if (
+            featureLimitResult.currentFeaturesCount +
+                featureLimitResult.newFeaturesCount >
+            featureLimitResult.limit
+        ) {
+            errors.push({
+                message: `We detected you want to create ${featureLimitResult.newFeaturesCount} new features in a project with ${featureLimitResult.currentFeaturesCount} current features and a limit of ${featureLimitResult.limit}`,
```

Comment:
Needed to read this 3 times, something is off. Better wording

We detected you want to create ${featureLimitResult.newFeaturesCount} new features to a project that already has ${featureLimitResult.currentFeaturesCount} existing features, exceeding the maximum limit of ${featureLimitResult.limit}.